test/e2e/specs/specs-playwright/wp-blocks__coblocks-gutter-spec.ts

```diff
@@ -1,3 +1,8 @@
+/**
```

Comment:
Does this need to be at the top of the file? (I guess the answer is 'yes').

What would happen if someone moves this below the `import` section? Is there some linting we can do to ensure it doesn't happen?

Reply:
Huh, I just tried this out actually, and it doesn't seem to care! It seems like you can put it anywhere (except of course over a function declaration, etc.) and it will find it! I tried below import statements, and even at the very end of the file - it found it in all cases!

Reply:
It might still be nice to nudge people to always have it in the same spot though for readability/consistency - I'll see if/how that can be enforced in linting.

Reply:
Spun this up in #56102 to look at later as it will take a bit of investigation of the best way to do this.